### ISSUE TITLE
Feature Request: Reveal Seed Phrase

### ISSUE THREAD

**Issue by pseudozach** [jamaljsr/polar#477]
**Is your feature request related to a problem? Please describe.**
I'm trying to test lnurl-auth across different implementations both within polar and other mobile wallets. For this I need to make sure same seed phrase ends up with same lnurl-auth signature.

**Describe the solution you'd like**
It would be great if seed phrase is saved in a file or echoed to log file, or even better if it can be a button on the GUI to reveal it.

**Additional context**
Thanks for this great tool, it's a joy to use!

**Comment by jamaljsr** [comment]
Hey @pseudozach, for LND we use the `--noseedbackup` config flag to autogenerate a seed and automatically unlock the node on startup. You could try removing this flag in the Advanced Options of the node. Since the node will not unlock automatically, the UI will show an error because it thinks the node failed to startup. But you would still be able to interact with the node via the CLI. 

Example:
```
$ docker-compose -f ~/.polar/networks/1/docker-compose.yml exec alice lncli unlock 
```

**Comment by pseudozach** [comment]
thanks I realized that --noseedbackup is used after I opened the issue. However I wasn't able to manually generate a wallet and unlock it, docker just keeps crashing..

I guess now with lnd v0.13 there's an alternative method to have a seed phrase and also auto unlock with a file but that might be a bit of an overkill for this. I've since found another way to accomplish what I wanted to do, so feel free to close the issue.
Thanks again!
